### PR TITLE
Create check report mistakes module for coach, little fix

### PR DESCRIPTION
**Pull request by ExxZzyy**
Create check report mistakes module for coach, little fix

### CODE REVIEW

File: src/app/services/store/coach-topic-store.service.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class CoachTopicStoreService {
       });
   }
 
-  getTopic(id: string) {
+  getTopic(id: string | undefined) {
```
[the remark this answers is not in the document]
Comment (ExxZzyy) [reply src/app/services/store/coach-topic-store.service.ts]: Done 457de71a7f55117cbdfc3dd340b16f92b7fdb417

**Comment by RomPavel** [comment]
image

**Review by shuliakleonid — COMMENTED** [review]
All done

**Comment by ExxZzyy** [comment]
Removed number from coach name d436f55db59de4f2811f2116e134809bbf422ad8

File: src/app/models/coach-profile-editor/topic-adding-editing-dialog/topic-adding-editing-dialog.component.scss [hunk 2]
```diff
@@ -24,3 +24,7 @@ h3 {
   float: right;
   margin-bottom: 30px;
 }
+
+.description {
+  color: darkred;
```
Comment (tsiananel) [inline src/app/models/coach-profile-editor/topic-adding-editing-dialog/topic-adding-editing-dialog.component.scss:29]: Let use var from '../../../theme/variables';  $error  =  darkred

**Review by tsiananel — APPROVED** [review]

File: src/app/models/coach-profile-editor/topic-adding-editing-dialog/topic-adding-editing-dialog.component.scss (hunk 2 above)
Comment (ExxZzyy) [reply src/app/models/coach-profile-editor/topic-adding-editing-dialog/topic-adding-editing-dialog.component.scss]: 855095f37e02e1f1bb3c87d82b109df2fc36eadc

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**